kuzzleio/documentation-V2
# KZL-985 remove js sdk from deps

## What does this PR do?
Remove sdk-javascript from dependencies because this is not a direct dependency, it's only used to run snippets tests and not to build the documentation website.

### How should this be manually tested?
run test locally and check in `package.json` if the `kuzzle-sdk` deps was not added.

### Boyscout
Fix linter error : Missing semicolon
